### DISCUSSION TITLE
Increase of text and button size for touch screen?

### DISCUSSION THREAD

**Discussion by hartmutgw2** [winder/Universal-G-Code-Sender#2279] [Help]
Hello,

I like the functionality of UGS but text and buttons are very small for touch screen operation - tried with 7" and 11" touch screens.
Unfortunately I was not able to figure out how to tweak the interface.
Any suggestions for improving touch screen operation?

Hartmut

**Comment by breiler** [comment]
What resolutions are those screens?
What OS are you running?
Which variant of UGS (platform or classic)?

Normally the font sizes (which also determines the button sizes) are fetched from the operating system. I noticed that some buttons that we have custom made does not scale properly. 

But generally, if you want bigger buttons you need to adjust the theme or scaling in your OS.

**Reply by hartmutgw2** [reply]
I'm using ugsplatform 2.18 on RPI 3 running Raspberry OS bullseye.
The current screen is a Waveshare WS-16642 11.6" with native resolution of 1920×1080.
I tried various resolutions in /boot/config.txt and xrandr like 1920x1080, 800x600 and 640x480.
But maybe need to find out how to scale - that's a good hint.

Thanks!

**Comment by hartmutgw2** [comment]
@breiler 
I was able to improve the button size by using `xrandr ... --scale 0.5x0.5` but still need to figure out how to increase the font size.
BTW, I don't use a desktop and UGS is started from .xinitrc and maybe I need to add something like xsettingd or xset there.

here a snapshot with part from Jog Controller - the up/down arrows are tiny and difficult to hit:
  
![grafik](https://github.com/winder/Universal-G-Code-Sender/assets/141274108/e7426d43-72b8-4d65-889c-9737ada8035e)
